packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1390,7 +1390,7 @@ abstract class RenderBox extends RenderObject {
         } else {
           debugTimelineArguments = <String, String>{};
         }
-        debugTimelineArguments!['intrinsics dimension'] = describeEnum(dimension);
+        debugTimelineArguments!['intrinsics dimension'] = dimension.name;
```

Comment:
We should probably deprecate describeEnum and redirect people to the `name` getter (not necessary in this PR).

Reply:
I want to do this, but you need to approve the 4 step conversion: https://github.com/flutter/engine/pull/40571